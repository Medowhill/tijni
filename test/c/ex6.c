int *f() {
  int x;
  int *p = &x;
  return p;
}
