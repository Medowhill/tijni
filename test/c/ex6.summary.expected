{
  "f": {
    "params": [],
    "ret": [ "Allocsite(0)" ],
    "env": { "%x(f)": [ "Allocsite(0)" ] }
  }
}